src/test/java/com/microservicesteam/nutaxi/NutaxiRouteServiceApplicationIntegrationTest.java

```diff
@@ -63,18 +66,24 @@ public void setUp() {
 
     @Test
     public void shouldReturnWithRouteWhenGoogleDirectionIsPresent() throws Exception {
-        when(mockRouteService.getRoute(anyObject())).thenReturn(Optional.of(route()));
+        when(mockRouteService.getRoute(anyObject())).thenReturn(Optional.of(route(EXAMPLE_POLYLINE)));
 
         mockMvc.perform(get("/api/route")
                 .accept(APPLICATION_JSON)
                 .param("origin", "Budapest Futó u. 47")
```

Comment:
This is very ugly in the URL's encoded form. Could we use simple HUN address (no special character).